scripts/analyze.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/sh
+
+###################################################################
+# Script to batch-analyze Black Duck component licenses against 3rd
+# party metadata.
+#
+# Mandatory $1 Blackduck project
+# Mandatory $2 Blackduck version
+# Mandatory $3 filename
+# Optional $4 parameter is the URL of a BOM-Base server
+#
+# Recipe of this script:
+#
+# The listed project versions are extracted from Black Duck as SPDX
+# files and a tree of Package URLs. The tree of Package URLs is used
+# to create a second SPDX file using independent metadata from ClearlyDefined
+# and package management repositories. The declared licenses from both
+# sources are compared, resulting in a list of all packages and a diff list of
+# packages where the declared licenses don't match.
+###################################################################
+set -e
+
+function info () {
+  echo " +================================================================================================ "
+  echo " | ${1}"
+  echo " +------------------------------------------------------------------------------------------------ "
+}
+
+function checkEnvironmentVariables () {
+  info "Check Environment Variables"
+  if [ -z $BLACKDUCK_API_TOKEN ]; then
```

Comment:
```suggestion
  if [ -z "$BLACKDUCK_API_TOKEN" ]; then
```
Refer to https://github.com/koalaman/shellcheck/wiki/SC2068